rosbridge_library/test/internal/test_ros_loader.py

```diff
@@ -33,14 +29,10 @@ def test_bad_msgnames(self):
             "\\",
         ]
         for x in bad:
-            self.assertRaises(
-                ros_loader.InvalidTypeStringException, ros_loader.get_message_class, x
-            )
-            self.assertRaises(
-                ros_loader.InvalidTypeStringException,
-                ros_loader.get_message_instance,
-                x,
-            )
+            # TODO: Remove InvalidModuleException
```

Comment:
I thought I fixed this already, but I guess I missed one..can you fix this too?

I changed the `bad` strings because the type string parsing changed for ROS 2 to support the `a_msgs/msg/B` format.

Reply:
Unfortunately, it couldn't.

```sh
install/rosbridge_library/lib/python3.8/site-packages/rosbridge_library/internal/ros_loader.py:168: InvalidModuleException
============================================================================================================================================================================================ short test summary info =============================================================================================================================================================================================
FAILED rosbridge_library/test/internal/test_ros_loader.py::TestROSLoader::test_bad_msgnames - rosbridge_library.internal.ros_loader.InvalidModuleException: Unable to import not.msg from package not. Caused by: No module named 'not'
========================================================================================================================================================================================== 1 failed, 15 passed in 0.32s ==========================================================================================================================================================================================
```

Reply:
Can I remove this part? Then I can remove `InvalidModuleException`.

```
            "not/better/still",
            "not//better//still",
            "not///better///still",
```

Reply:
Removed in 6dc176125f36e81cb5cb825bcd98a4250b6ed75a as a test.